.travis.yml

```diff
@@ -1,14 +1,14 @@
 language: cpp
-dist: trusty
+dist: xenial
```

Comment:
why did you change the trusty to xenial?

Reply:
As explained in the commit, Meson 0.47 requires Python 3.5.2 which is not shipped in Trusty.

Reply:
why meson must upgrade to 0.47?

Reply:
The `feature` option type was added in Meson 0.47, as implied in the commit message.